test/presenters/base.presenter.test.js

```diff
@@ -57,4 +57,40 @@ describe('Base presenter', () => {
       expect(result).to.equal('0000123')
     })
   })
+
+  describe('_signedCreditValue method', () => {
+    it('returns a negative value if given a credit', async () => {
+      const presenter = new BasePresenter()
+
+      const result = presenter._signedCreditValue(123, true)
+
+      expect(result).to.equal(-123)
+    })
+
+    it('returns a negative value if given a debit', async () => {
+      const presenter = new BasePresenter()
+
+      const result = presenter._signedCreditValue(123, false)
+
+      expect(result).to.equal(123)
+    })
+  })
```

Comment:
If my previous suggestion is taken on this would become

```suggestion
  describe('_reverseSignOfValue method', () => {
    it('returns a negative value if given a positive value', async () => {
      const presenter = new BasePresenter()

      const result = presenter._reverseSignOfValue(123)

      expect(result).to.equal(-123)
    })

    it('returns a positive value if given a negative value', async () => {
      const presenter = new BasePresenter()

      const result = presenter._reverseSignOfValue(-123)

      expect(result).to.equal(123)
    })
  })
```